Source/Core/Core/IOS/ES/ES.h

```diff
@@ -92,8 +78,8 @@ class ES final : public Device
 
     u16 gid = 0;
     u32 uid = 0;
+    // The same context is used for both title imports and exports.
     TitleImportContext title_import;
```

Comment:
Not sure if leaving this as Title _Import_ Context makes too much sense when its also used for Export, but I'm not really convinced that simply calling it `TitleContext` (inside a class `Context`) is a sensible choice either. Can be ignored if we can't come up with a better name...